http_client/src/main/java/com/fitgoal/BaseClient.java

```diff
@@ -0,0 +1,83 @@
+package com.fitgoal;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dropwizard.jackson.Jackson;
+import okhttp3.HttpUrl;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+import javax.inject.Inject;
+import java.io.IOException;
+import java.util.Arrays;
+
+import static java.net.HttpURLConnection.HTTP_BAD_REQUEST;
+
+abstract class BaseClient {
+
+    private static final MediaType JSON_MEDIA_TYPE = MediaType.parse("application/json; charset=utf-8");
+    private static final int UNPROCESSABLE_ENTITY_STATUS_CODE = 422;
```

Comment:
Its OK, but fyi there is a class for all HTTP statuses called `org.eclipse.jetty.http.HttpStatus`
![image](https://user-images.githubusercontent.com/11231875/68766010-bdbc9c80-0626-11ea-837d-5ccd5c881bde.png)

Reply:
Refactored. Thanks!